### PR TITLE
better translations for the french language

### PR DESCRIPTION
**Pull request by kevincaradant**
Hi

Maybe it's not yet perfect but I think it's more understandable now :)

### CODE REVIEW

**Comment by garfield69** [comment]
Terrific, thank you for this. Yeah, Travis is having a little fit, not related to your contribution.

File: _locales/fr_fr.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "ABOUT/bugs/p2": " . Je pourrai ainsi le suivre.",
     "ABOUT/contributing/hdr": "contribuer ",
     "ABOUT/contributing/link": "forker DuckieTV et soumettre vos pull requests ",
-    "ABOUT/contributing/p1": "Vous souhaitez contribuer à DuckieTV et tremper dans Angular.js ? N'hésitez pas à ",
+    "ABOUT/contributing/p1": "Vous souhaitez contribuer à DuckieTV et vous impliquez dans Angular.js ? N'hésitez pas à ",
```
Comment (kevincaradant) [inline _locales/fr_fr.json:8]: Ah sorry @garfield69, here, that should be `vous impliquer` and not `vous impliquez`. I let you fix that in a commit in your side ?

File: _locales/fr_fr.json [hunk 2]
```diff
@@ -115,10 +115,10 @@
     "COMMON/global-size-min/hdr": "Taille minimale",
     "COMMON/global-size/alert": "Entrez la taille en nombre de mégaoctets.",
     "COMMON/global-size/form": "MB:",
-    "COMMON/ignore-global-quality/lbl": "Ignorer la qualité de ce spectacle",
-    "COMMON/ignore-ignore-keywords/lbl": "Ignorer l'exclusion pour ce spectacle",
+    "COMMON/ignore-global-quality/lbl": "Ignorer la qualité de cette série",
+    "COMMON/ignore-ignore-keywords/lbl": "Ignorer le mots-clés EXCLUS pour la série",
```
Comment (kevincaradant) [inline _locales/fr_fr.json:119]: les mots instead of le mots. Sorry I'm tired :)

File: _locales/fr_fr.json [hunk 3]
```diff
@@ -214,9 +214,9 @@
     "EPISODEDOWNLOADEDjs/is-downloaded/lbl": "Marqué comme téléchargé",
     "EPISODEDOWNLOADEDjs/not-downloaded/lbl": "Marquer cet épisode comme téléchargé",
     "EPISODEWATCHEDjs/is-marked/lbl": "Regardé le ",
-    "FASTSEARCH/episodes-favorites/hdr": "Épisodes de favoris",
+    "FASTSEARCH/episodes-favorites/hdr": "Épisodes tirés de vos favoris",
     "FASTSEARCH/hdr": "Recherche rapide",
-    "FASTSEARCH/series-favorites/hdr": "Série de favoris",
+    "FASTSEARCH/series-favorites/hdr": "Série tirés de vos favoris",
```
Comment (kevincaradant) [inline _locales/fr_fr.json:219]: Séries tirées instead of Série tirés

**Comment by garfield69** [comment]
don't worry about it today. You can fix it later and submit a new PR when you are ready.

**Pull request merged**